app/views/users/phone_setup/index.html.erb

```diff
@@ -2,19 +2,11 @@
 
 <%= render(VendorOutageAlertComponent.new(vendors: [:sms, :voice])) %>
 
-<%= image_tag asset_url('2FA-voice.svg'), alt: '', width: 200, height: 113, class: 'margin-bottom-2' %>
-
 <%= render PageHeadingComponent.new.with_content(t('titles.phone_setup')) %>
 
 <p>
   <%= t('two_factor_authentication.phone_info_html') %>
```

Comment:
Qustion: In the design, "each time you sign in" is unbolded. Should we be unbolding that here?

Reply:
yea we can, I left it as is cause idk why it got bolded in the first place but i might be able to clean that up.